CODEOWNERS

```diff
@@ -58,6 +58,7 @@ k8s/**/common/ia/case-documents-api.yaml @hmcts/immigration-asylum
 k8s/**/common/ia/case-notifications-api.yaml @hmcts/immigration-asylum
 
 ## Divorce
+k8s/demo/common/divorce/*.yaml @hmcts/divorce
```

Comment:
what does it do?

Reply:
Allows us to review our team to review all files matching `k8s/demo/common/divorce/*.yaml` without DevOps approval (in the future)